app/admin/landing_theme.rb

```diff
@@ -125,6 +126,7 @@ def update
         ls.input :subject, as: :ajax_select, data: { url: :admin_subjects_path, search_fields: [:label] }
         ls.input :description, as: :quill_editor
         ls.input :description_explanation, as: :quill_editor
+        ls.input :description_prefill
```

Comment:
```suggestion
        ls.input :description_prefill, as: :quill_editor
```
Pour avoir l'éditeur wyswyg

Reply:
Justement, je veux pas l'éditeur :) Car le champs final, côté visiteur, y'a pas de wysiwyg